package.json

```diff
@@ -11,8 +11,8 @@
   },
   "dependencies": {
     "find-cache-dir": "^3.3.1",
+    "lmdb": "^2.2.4",
```

Comment:
The `package.json` changes are part of #932, can you revert them and update the lockfile accordingly for this PR?